### ISSUE TITLE
[UX/UI] Bookreader scroll speed too fast on iOS tablet/Safari

### ISSUE THREAD

**Issue by UcDust** [ucd-library/dams#266]
From testing https://dev.dams.library.ucdavis.edu/item/ark:/87287/d7kw8c, this was mentioned:
_scroll speed between pages is crazy fast.  it only gets in about 1 or 2 frames of the scroll.._

**Comment by wrenaria** [comment]
@UcDust Yeah, the scroll animation isn't really adding anything to this experience. I'm fine with disabling it.
